### DISCUSSION TITLE
Inheritance for objectbricks only after saving children

### DISCUSSION THREAD

**Discussion by paulhaem** [pimcore/pimcore#16433] [Q&A / Support]
Steps to reproduce
1. Add objectbricks with localized fields to a parent data object
2. Child shows inherited values in the admin panel
3. Inherited values are not present using PHP 
4. Save the children 
5. Inherited values are now present using PHP 

I am not sure if this is an issue on how I access the objectbricks values or if this is the standard behaviour when working with inherited values. At least its not what one would expect. Would be glad if someone could shed some light on this.

**Comment by fashxp** [comment]
sounds not right to me neither. maybe you can dig a bit deeper. thanks...
